src/beta_inc.jl

```diff
@@ -266,7 +266,7 @@ function beta_inc_asymptotic_symmetric(a::Float64, b::Float64, lambda::Float64,
     b0 = zeros(22)
     c = zeros(22)
     d = zeros(22)
-    e0 = 2/sqrt(pi)
+    e0 = 2/Float64(IrrationalConstants.sqrtπ)
```

Comment:
Why is the `Float64` necessary?  `Int / Irrational` should already result in `Float64`

Reply:
It's not necessary, I just wanted to be mote explicit and make it clearer that `e0` is a `Float64`. Would you like me to change it?

Reply:
yes, please change

Reply:
```suggestion
    e0 = 2/IrrationalConstants.sqrtπ
```